thundra/integrations/botocore.py

```diff
@@ -58,7 +57,9 @@ def before_call(self, scope, wrapped, instance, args, kwargs, response, exceptio
         if 'Item' in self.request_data:
             self.escape_byte_fields(self.request_data['Item'])
 
-        self.OPERATION.get(operation_name, dummy_func)(scope)
+        # DB statement tags should not be set on span if masked
+        if not config.dynamodb_statement_masked():
+            self.OPERATION.get(operation_name, dummy_func)(scope)
```

Comment:
I know it has not introduced with this PR but I cannot understand role of the `dummy_func` here?

Reply:
### I think it is given to handle the case operation name is not in operation dictionary, to not call None as function. I can remove dummy_func and and just add a membership check to refactor?